### PR TITLE
Add a target for pikkr

### PR DESCRIPTION
**Pull request by dtolnay**
https://github.com/pikkr/pikkr

### CODE REVIEW

**Comment by dtolnay** [comment]
What do we do about the Travis build? Pikkr requires AVX instructions which Travis does not expose.

File: pikkr/parse.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#![no_main]
+
+#[macro_use] extern crate libfuzzer_sys;
+extern crate pikkr;
+
+fuzz_target!(|data| {
+    let q = vec!["$.x".as_bytes()];
```
Comment (killercup) [inline pikkr/parse.rs]: Would it make sense to also fuzz the query parser? Either as a new target, or by splitting `data` into a query and an input part

Comment (dtolnay) [reply pikkr/parse.rs]: Possibly later. I would wait until the bugs exposed by this target have been fixed, then look at whether fuzzing the query parser discovers anything new. For now this target will tend to give easier repro cases.

Also the input JSON is much more likely to be untrusted than the query string in typical usage, so even if the query parser is buggy, most people may be comfortable using the library anyway.

**Comment by frewsxcv** [comment]
If you want, feel free to add the bugs you found with this fuzz target to the trophy case https://github.com/rust-fuzz/trophy-case

**Comment by killercup** [comment]
Can you give it quick rebase? https://github.com/pikkr/pikkr/pull/65 has landed which allows AVX-less builds

**Comment by dtolnay** [comment]
Rebased. I kept the RUSTFLAGS bit so that on an AVX2 capable machine I can run it with the feature enabled.

**Comment by killercup** [comment]
Awesome, thanks!

bors r+

**Pull request merged**